extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -115,10 +115,18 @@ const DOUBLECLICK_BASE_URL =
     'https://securepubads.g.doubleclick.net/gampad/ads';
 
 /** @private @enum {number} */
-const RTC_ATI_ENUM = {
+export const RTC_ATI_ENUM = {
   RTC_SUCCESS: 2,
   RTC_FAILURE: 3,
 };
+RTC_ATI_ENUM[RTC_ERROR_ENUM.MALFORMED_JSON_RESPONSE] = 4;
+RTC_ATI_ENUM[RTC_ERROR_ENUM.DUPLICATE_URL] = 5;
+RTC_ATI_ENUM[RTC_ERROR_ENUM.INSECURE_URL] = 6;
+RTC_ATI_ENUM[RTC_ERROR_ENUM.MAX_CALLOUTS_EXCEEDED] = 7;
+RTC_ATI_ENUM[RTC_ERROR_ENUM.NETWORK_FAILURE] = 8;
+RTC_ATI_ENUM[RTC_ERROR_ENUM.UNKNOWN_VENDOR] = 9;
+RTC_ATI_ENUM[RTC_ERROR_ENUM.TIMEOUT] = 10;
+RTC_ATI_ENUM[RTC_ERROR_ENUM.MACRO_EXPAND_TIMEOUT] = 11;
```

Comment:
Having two Enums that differ just in whether their underlying representation is a string or a number is pretty awkward.  Among other things, it means we have to keep these two parts of the code in sync, or else we'll fall back to `RTC_FAILURE`. (Do we expect any legitimate uses of `RTC_FAILURE` after this PR, or will any represent cases where `RTC_ATI_ENUM` is incomplete?)

We can't just use `RTC_ERROR_ENUM` values straight because those are strings and it's more efficient to send a number, right?

Could `RTC_ERROR_ENUM` switch to using number values?  And then we could just always use `RTC_ERROR_ENUM` everywhere?

Reply:
Done